tutorials/terraform-quickstart/index.mdx

```diff
@@ -9,14 +9,14 @@ categories:
   - instances
   - terraform
   - elastic-metal
-tags: compute Terraform Scaleway-infrastructures cloud Elastic-Metal Instance HashiCorp-Terraform
+tags: Terraform Elastic-Metal Instances HashiCorp
 hero: assets/scaleway_terraform.webp
 dates:
   validation: 2024-04-22
   posted: 2018-04-06
 ---
 
-HashiCorp [Terraform](http://terraform.io) is an open-source software tool to deploy **IaaC**: Infrastructure as Code. It means that you can automate infrastructure resources such as Network, Instances, Elastic Metal servers and more. It allows you to use declarative configuration files to manage the full lifecycle — create new resources, manage existing ones, and delete those no longer needed. The configuration language used by Terraform is called [Hashicorp Configuration Language (HCL)](https://github.com/hashicorp/hcl).
+HashiCorp [Terraform](http://terraform.io) is an open-source software tool to deploy **IaaC**: Infrastructure as Code. It means that you can automate infrastructure resources such as Networks, Instances, Elastic Metal servers, and more. It allows you to use declarative configuration files to manage the full lifecycle — create new resources, manage existing ones, and delete those no longer needed. The configuration language used by Terraform is called [Hashicorp Configuration Language (HCL)](https://github.com/hashicorp/hcl).
```

Comment:
I think maybe Network was right, to describe the generic product category rather than a specific countable product. Or else maybe networks lower case?

Reply:
Ok. Thanks!